### DISCUSSION TITLE
Can I use both only Bootstrap-v3 CSS and bootstrap-v4 JS in same HTML file in angular

### DISCUSSION THREAD

**Discussion by ajaysharma92** [twbs/bootstrap#30823] [General] [answered]
link href="bootstrap.min.css" -----  This is BS-3 CSS file
script src="bootstrap.min.js"  -----  This is BS-4 JS file

can we import both files in same HTML file with different BS versions of CSS & JS?

**Comment by mdo** [comment, marked as answer]
No, the changes are too great. You'll need to use v4's JS with v4's CSS.

**Reply by ajaysharma92** [reply]
Thanks for your Answer
